code/modules/materials/geology/strata_permafrost.dm

```diff
@@ -6,5 +6,14 @@
 		/decl/material/solid/ice/lukrite,
 		/decl/material/solid/ice/rubenium,
 		/decl/material/solid/ice/trigarite,
-		/decl/material/solid/ice/ediroite
+		/decl/material/solid/ice/ediroite,
```

Comment:
Might be worth splitting some of these changes off into separate PRs to make this one more manageable for you.

Reply:
Not too sure if it would really make things more manageable really? This and the ices are pretty much intertwined together.